src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -138,10 +143,18 @@ public Person toModelType() throws IllegalValueException {
             throw new IllegalValueException(Weight.MESSAGE_CONSTRAINTS);
         }
         final Weight modelWeight = new Weight(weight);
+        if (bloodType == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    BloodType.class.getSimpleName()));
+        }
+        if (!BloodType.isValidBloodType(bloodType)) {
+            throw new IllegalValueException(Weight.MESSAGE_CONSTRAINTS);
```

Comment:
Shouldn't it be `BloodType.MESSAGE_CONSTRAINTS` instead of `Weight.MESSAGE_CONSTRAINTS`?